packages/app/src/components/VisualizationOptions/Options/HideTitle.js

```diff
@@ -83,10 +87,11 @@ const hideTitleSelector = createSelector([sGetUiOptions], uiOptions =>
 const mapStateToProps = state => ({
     visualizationType: sGetUiType(state),
     value: hideTitleSelector(state),
+    title: sGetUiOptions(state).title,
 })
 
 const mapDispatchToProps = dispatch => ({
-    onChange: enabled => dispatch(acSetUiOptions({ hideTitle: enabled })),
+    onChange: value => dispatch(acSetUiOptions(value)),
```

Comment:
I would prefer if this dispatch function took in `hideTitle` and `title` and constructed it's own parameter object `{hideTitle: X, title, Y}` instead of a pre-constructed object `value`, as it's currently exposing the inner structure of `acSetUiOptions` to the class.